README.md

```diff
@@ -39,12 +39,13 @@ ParadeDB is currently in Public Beta. Star and watch this repository to get noti
 - [ ] Analytics
   - [x] An analytical query engine over any object store or table format with [pg_lakehouse](https://github.com/paradedb/paradedb/tree/dev/pg_lakehouse#overview)
   - [ ] Column-oriented table access method for fast analytics inside Postgres
-  - [ ] High-volume data/Kafka ingest
 - [x] Self-Hosted ParadeDB
-  - [x] Docker image based on [bitnami/postgresql](https://hub.docker.com/r/bitnami/postgresql) & [deployment instructions](https://docs.paradedb.com/deploy/aws)
-  - [x] Kubernetes Helm chart & [deployment instructions](https://docs.paradedb.com/deploy/helm)
+  - [x] Docker image based on [postgres](https://hub.docker.com/_/postgres) & [deployment instructions](https://docs.paradedb.com/deploy/aws)
```

Comment:
nit: capital P

Reply:
I will bring this in the next PR, to avoid re-running the whole CI here